Fix build. Some parts of this patch are borrowed from ArchLinux gtk-chtheme
package:
https://github.com/archlinux/svntogit-community/blob/packages/gtk-chtheme/trunk/PKGBUILD#L20
--- gtk-chtheme-0.3.1/Makefile  2004-02-09 18:08:41.000000000 +0100
+++ gtk-chtheme-0.3.1-patch/Makefile    2021-08-26 09:24:10.832781503 +0200
@@ -4,7 +4,7 @@

 LDFLAGS = $(shell pkg-config --libs gtk+-2.0)
 CFLAGS += -Wall
-CFLAGS += $(shell pkg-config --cflags gtk+-2.0) -DGTK_DISABLE_BROKEN -DGTK_DISABLE_DEPRECATED
+CFLAGS += $(shell pkg-config --cflags gtk+-2.0)
 CFLAGS += -DPROJNAME='"$(PROJNAME)"' -DVERSION='"$(VERSION)"'
 CPPFLAGS =
 CXXFLAGS =
@@ -38,7 +38,6 @@
    -rm -f $(EXENAME).spec $(EXENAME).1 Makefile.bak

 install: all
-   strip      $(EXENAME)
    install -d              $(DESTDIR)/$(BINDIR)
    install -c $(EXENAME)   $(DESTDIR)/$(BINDIR)
    install -d              $(DESTDIR)/$(MANDIR)/man1
